sonar-cxx-plugin/src/test/java/org/sonar/plugins/cxx/TestUtils.java

```diff
@@ -171,16 +172,16 @@ public static void addInputFile(DefaultFileSystem fs,
         temp.setCharAt(0, Character.toUpperCase(temp.charAt(0)));
         path = temp.toString();
       }
-      inputFile = new DefaultInputFile(path);
-      inputFile.setAbsolutePath(path);
+      inputFile = new DefaultInputFile("key", path);
```

Comment:
for reference
https://github.com/SonarSource/sonarqube/blob/0ae5ddd039e29094eee818370fe34810d3a225f7/plugins/sonar-xoo-plugin/src/test/java/org/sonar/xoo/lang/SyntaxHighlightingSensorTest.java